modules/ossm-applying-certificates-to-multi-cluster-topology.adoc

```diff
@@ -0,0 +1,114 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-applying-certificates-to-multi-cluster-topology_{context}"]
+= Applying certificates to a multi-cluster topology 
+
+This procedure explains how to apply security certificates to a multi-cluster topology.
+
+.Prerequisites
+
+* You have access to two {ocp-product-title} clusters with external load balancer support. 
+
+* You have installed the {SMProduct} 3.0 Operator to two clusters.
```

Comment:
Instead of "two", should this refer to the two clusters mentioned in the previous bullet? Something like this:
```suggestion
* You have installed the {SMProduct} 3.0 Operator to these clusters.
```

Reply:
done